src/main.js

```diff
@@ -6,7 +6,7 @@ import VueAxios from 'vue-axios';
 import GAuth from 'vue-google-oauth2';
 import App from './App.vue';
 import router from './router';
-import store from './store';
+import store from './store/index';
```

Comment:
There is no need for '/index' part. It is default file name to load data if it exists.

Reply:
Corrected. Should I merge ?